cdn-examples/server/auth0/DX/README.md

```diff
@@ -0,0 +1,23 @@
+
+## Configuration
+This worker includes several environment variables that can be setup to customise the behaviour of the worker.
+
+| Variable              | Description                                                                          | Default        | Example Format                              |
+| --------------------- | ------------------------------------------------------------------------------------ | -------------- | ------------------------------------------- |
+| publicKey             | The Arkose public key to use in this worker, available via the portal                |                | `11111111-1111-1111-1111-111111111111`      |
+| privateKey            | The Arkose private key to use in this worker, available via the portal               |                | `11111111-1111-1111-1111-111111111111`      |
+| clientSubdomain       | Client subdomain name provided to you via the Arkose Labs team                       | `client-api`   |                                             |
+| verifySubdomain       | Verify subdomain name provided to you via Arkose Labs team                           | `verify-api`   |                                             |
+| errorUrl              | A page to route the user to when an error occurs (outside of fail open)              |                | `"https://www.arkoselabs.com"`              |
+| arkoseCookieLife      | The length of time the Arkose cookies should be active for (in milliseconds)         |                | `300000`                                    |
+| arkoseCookieName      | The name of the cookie that the Arkose token will be stored in                       |                | `arkoseToken`                               |
+| arkoseErrorCookieName | The name of the cookie that an Arkose error will be stored in                        |                | `arkoseError`                               |
+| scriptMaxRetryCount   | The number of times to retry resetting Arkose if there is a client side error        |                | `3`                                         |
+| verifyMaxRetryCount   | The number of times to retry verification if there is an error                       |                | `3`                                         |
+| failOpen              | A boolean to indicate if we should fail open or not in case of outage                |                | `true`                                      |
+| tokenEnforcement      | A Boolean to indicate whether to handle a failure on Arkose. Can be useful to help initial setup by setting to false.        | `true`                                 |
```

Comment:
I think I missed this in the other repo, but there's some inconsistent casing on the Boolean here